src/assignment/assignment.helper.tsx

```diff
@@ -70,13 +65,17 @@ export class AssignmentHelper {
 				user
 			);
 
-			setCurrentAssignment({});
-			setLoadingInfo({ state: 'loading' });
-
-			navigate(history, APP_PATH.ASSIGNMENT_EDIT.route, { id: duplicatedAssignment.id });
+			const newWindow = window.open(
+				buildLink(APP_PATH.ASSIGNMENT_EDIT.route, { id: duplicatedAssignment.id }),
+				'_blank',
+				'noopener,noreferrer'
+			);
+			if (newWindow) {
+				newWindow.opener = null;
+			}
```

Comment:
Is dit de eerste instantie van een `open-in-new-tab`-mechanisme? Ik zou het alvast als generieke util opdelen